core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -55,7 +55,11 @@ enum class UniqueTarget(val inheritsFrom:UniqueTarget?=null) {
     }
 }
 
-enum class UniqueType(val text:String, vararg targets: UniqueTarget) {
+enum class UniqueFlag {
+    HideInCivilopedia,
+}
+
+enum class UniqueType(val text:String, vararg targets: UniqueTarget, val flags: List<UniqueFlag> = emptyList()) {
```

Comment:
List? I had a discrete Boolean in mind, but if we want to be extensible, this is what EnumSet was created for, even if it is more legacy Java than idiomatic Kotlin... As always, just food for thought.